docs/docs/02-configuration/04-enum.mdx

```diff
@@ -49,6 +52,22 @@ public partial class CarMapper
 }
 ```
 
+Applied to an explicit enum value pair mapped by this method.
+Attribute is only valid on mapping method with enums as parameters and
+configured with the `ByName` strategy.
+
+```csharp
+[Mapper]
+public partial class CarMapper
+{
+    [MapEnum(EnumMappingStrategy.ByName, IgnoreCase = true)]
+    // highlight-start
+    [MapEnumValue(CarMake.Datsun, CarMakeDto.Nissan)]
```

Comment:
We try to avoid brand names in our documentation (eg. use `CarFeature => CarFeatureDto` with `CarFeature.AWD => CarFeatureDto.AllWheelDrive` instead). My bad I used car brands in the issue's description 😞